### PR TITLE
Add launcher URL to docs.

### PR DESCRIPTION
**Pull request by dlorenc**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,7 +2,10 @@
 
 [![Build Status](https://travis-ci.org/GoogleCloudPlatform/nodejs-docker.svg?branch=master)](https://travis-ci.org/GoogleCloudPlatform/nodejs-docker)
 
-This repository contains the source for the `gcr.io/google_appengine/nodejs` [docker](https://docker.com) image. This image can be used as the base image for running applications on [Google App Engine Managed VMs](https://cloud.google.com/appengine), [Google Container Engine](https://cloud.google.com/container-engine), or any other Docker host.
+This repository contains the source for the Google-maintained Node.js [docker](https://docker.com) image.
+This image can be found at `launcher.gcr.io/google/nodejs` or `gcr.io/google-appengine/nodejs`, and this image can be used as the base image
```
Comment (nkubala) [inline README.md]: are we just using `launcher.gcr.io/google...` and `l.gcr.io/google...` interchangeably?

Comment (dlorenc) [inline README.md]: Yeah they prefer we show launcher, then mention that l.gcr.io is a shortcut.

File: README.md [hunk 2]
```diff
@@ -2,7 +2,10 @@
...
+for running applications on [Google App Engine Managed VMs](https://cloud.google.com/appengine),
```
Comment (ofrobots) [inline README.md]: nit: while you're here, can you drop 'Managed VMs'

Comment (dlorenc) [inline README.md]: Done.

**Pull request merged**